File: src/Client/wwwroot/app/servicemodels/tournament.ts
import {Team} from './team';
import {TournamentResultItem} from './tournamentResultItem';
import {Round} from './round';


export class Tournament {
    id: string;
    creationDate: Date;
    table: TournamentResultItem[];
    rounds: Round[];
    logoUrl: string;

    constructor(public name: string, public participants: Team[]) { }
}

File: src/Tsc.Application/ServiceModel/Tournament.cs
using System;
using System.Collections.Generic;

namespace Tsc.Application.ServiceModel
{
    public class Tournament
    {
        public Guid Id { get; set; }

        public string Name { get; set; }

        public DateTime CreationDate { get; set; }

        public List<Team> Participants { get; set; }

        public List<TournamentResultItem> Table { get; set; }

        public List<Round> Rounds { get; set; }

        public string LogoUrl { get; set; }
    }
}
